lmfdb/characters/main.py

```diff
@@ -156,6 +164,24 @@ def common_parse(info, query):
     parse_ints(info, query, "modulus", name="modulus")
     parse_ints(info, query, "conductor", name="conductor")
     parse_ints(info, query, "order", name="order")
+    if 'inducing' in info:
+        try:
+            validate_label(info['inducing'])
+            parts_of_label = info['inducing'].split(".")
+            if len(parts_of_label) != 2 or not str.isalpha(parts_of_label[1]):
+                raise ValueError("Invalid character orbit label format, expected N.a")
+            primitive_modulus = int(parts_of_label[0])
+            primitive_orbit = class_to_int(parts_of_label[1])+1
+            if db.char_dirichlet.count({'modulus':primitive_modulus,'is_primitive':True,'orbit':primitive_orbit}) == 0:
+                print("hi")
+                raise ValueError("Primitive character orbit not found")
+            if 'conductor' in query:
+                flash_error("Conductor constraint %s ignored due to specification of primitive inducing character", info['conductor'])
```

Comment:
I've tended to treat incompatible constraints like this as giving no results rather than just ignoring them.  It does make parsing things harder though.

Reply:
See [this line](https://github.com/LMFDB/lmfdb/blob/5bd0fe0d6d5882a4e4fa81dd133d2a2416ca1ef1/lmfdb/number_fields/number_field.py#L925) for example, which I added a few days ago.

Reply:
It would be nice to have a general policy about when approach like this (a flash error that we're ignoring input because another input is more specific) is okay.

Reply:
> I've tended to treat incompatible constraints like this as giving no results rather than just ignoring them. It does make parsing things harder though.

I would normally do that too, but in this case you can't do that without inventing a constraint they did not enter (there is no column for primitive orbit label, and you can't specify two values for the conductor).

But thinking about it again I realize we could set `primitive_orbit` to zero, which would yield no results and not be visible to the user, I'll do that instead.

Reply:
I made a further change so that it never ignores constraints (if the induced label is invalid you get an error, if it is valid but incompatible with the conductor you get no results)